bcda/web/api_test.go

```diff
@@ -568,9 +720,9 @@ func bulkRequestHelper(endpoint, resourceType string) (string, func(http.Respons
 	} else {
 		endpoint = "Group/all"
 		if resourceType != "" {
-			requestUrl = fmt.Sprintf("/api/v1/%s/$export?_type=%s", endpoint, resourceType)
+			requestUrl = fmt.Sprintf("/api/v1/%s/$export?_type=%s%s", endpoint, resourceType, sinceRequestString)
 		} else {
-			requestUrl = fmt.Sprintf("/api/v1/%s/$export", endpoint)
+			requestUrl = fmt.Sprintf("/api/v1/%s/$export%s", endpoint, sinceRequestString)
```

Comment:
Same as above: no `?` to begin the query

Reply:
Good catch!